packages/core/src/app/payment/paymentMethod/HostedWidgetPaymentMethod.tsx

```diff
@@ -137,6 +137,7 @@ class HostedWidgetPaymentMethod extends Component<
             method,
             onUnhandledError = noop,
             setValidationSchema,
+            isPaymentDataRequired
```

Comment:
Can you please add this logic to [`HostedWidgetPaymentComponent`](https://github.com/bigcommerce/checkout-js/blob/master/packages/hosted-widget-integration/src/HostedWidgetPaymentComponent.tsx) as well please.